.github/pull_request_template.md

```diff
@@ -0,0 +1,48 @@
+<!--- PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][OSDOCS-<jira-issue-id>]: <short-description-of-the-pr> --->
+
+Version(s):
+<!--- Specify the version or versions of OpenShift your PR applies to.
+Open your PR against the `main` branch for if your changes apply to the latest releases. --->
+
+Issue:
+<!--- Add a link to the Bugzilla, Jira, or GitHub issue. --->
+
+Link to docs preview:
+<!--- Add direct link(s) to the exact page(s) with updated content from the preview build.
+  - The preview will be generated after you open the PR.
+  - You will need to edit the comment to add the link after the preview builds.
+  - Review the preview build to make sure your changes are rendering properly. --->
+
+Additional information:
+<!--- Optional: Include additional context or expand the description here.
+
+Next steps after opening your PR:
+
+* Ask for peer review from the OpenShift docs team:
+  - Ping the peer review squad (@peer-review-squad) requesting a review in the #forum-docs-review channel (CoreOS Slack workspace)
```

Comment:
The CoreOS Slack is technically internal, so wondering two things:

1. Should we not show a link to it?
2. Should we tell people who aren't internal an alternative option for requesting review? Like, if you're a Red Hatter, use slack, if not, ping the "@ openshift/team-documentation" in a comment? (just worded more nicely than that :wink:

Reply:
for my part(as a PR author), pinging a github team in a comment would always be my preferred workflow over opening a PR and then going to slack and tagging a team there, even if i have access to the slack channel.  

but i realize that not all individuals/teams monitor github notification emails religiously (in some cases because they simply get too many of them for things that aren't directly relevant to them)

Reply:
I tried to split the difference without adding too much ambiguity. I have added a section that community members can @ mention the docs team, but that RH associates will get a more timely response in the Slack channel.

Reply:
Yeah, there is way too much activity in the repo, so our response rates to email pings typically isn't as good as Slack pings, so that's why Slack is the preferred method from our side.